packages/backend/src/config/getDeliveryServiceProperties.ts

```diff
@@ -10,6 +10,11 @@ const DEFAULT_DELIVERY_SERVICE_PROPERTIES: DeliveryServiceProperties = {
     //100Kb
     sizeLimit: 100000,
     notificationChannel: [],
+    smtpHost: process.env.SMTP_HOST as string,
```

Comment:
If we use the config file the config should bot be loaded from process.env

Reply:
Then in default value shall I put it empty string?

Reply:
Yes